sky/shell/platform/ios/FlutterView.mm

```diff
@@ -30,4 +39,25 @@ - (BOOL)enableInputClicksWhenVisible {
   return YES;
 }
 
+/* The container itself is not accessible but contains accessible elements. */
+- (BOOL)isAccessibilityElement {
+  return NO;
+}
+
+- (NSArray*)accessibleElements {
+  return _accessibleElements;
+}
+
+- (NSInteger)accessibilityElementCount {
+  return [[self accessibleElements] count];
+}
+
+- (id)accessibilityElementAtIndex:(NSInteger)index {
+  return [[self accessibleElements] objectAtIndex:index];
```

Comment:
Also, you can subscripting here (after you do the bounds check of course). So something like `self.accesibleElements[index]`